CHANGELOG.md

```diff
@@ -60,6 +60,7 @@
 
 - The codebase now relies on the package `cryptonite` (instead of `ed25519`) for Ed25519 implementation (CO-325)
 
+- **[API BREAKING CHANGE]** Improve diagnostic for `NotEnoughMoney` error (CBR-461)
```

Comment:
Reminds me also of https://iohk.myjetbrains.com/youtrack/issue/CBR-449, heh.

Reply:
![image](https://user-images.githubusercontent.com/5680256/46419447-3536b180-c72e-11e8-8d97-eb4bf784c3c6.png)

Whoops :trollface: